### PR TITLE
Added a How To File GitHub Issues guide

### PR DESCRIPTION
**Pull request by RichardLitt**
See https://github.com/ipfs/go-ipfs/pull/2786

License: MIT
Signed-off-by: Richard Littauer <richard.littauer@gmail.com>

### CODE REVIEW

File: docs/github-issue-guide.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- "question: How do I use `ipfs block <hash>`?"
+
+Putting the command in backticks helps us parse the natural language description, and is generally suggested.
+
+This is a _living guide_. If you see anything that should be here and isn't, or have ideas on improvement, please open a "meta" issue.
+
+### Type
+
+- "bug": If what you are filing is a bug.
+- "Meta": If it is something about how we run go-ipfs, and not code related in itself.
```
Comment (whyrusleeping) [inline docs/github-issue-guide.md]: odd that meta is capitalised while the rest arent

File: docs/github-issue-guide.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+- "Linux"
+- "Windows"
+- "Mac"
+- Etc.
+
+### Processor
+
+Your processor.
+
+- "32"
```
Comment (whyrusleeping) [inline docs/github-issue-guide.md]:
```
- x86
- amd64
- arm
```

File: docs/github-issue-guide.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- "tools"
+- "tracking"
+- "unix vs dag"
+
+### Priority
+
+- "P0 - Operations Functioning"
+- "P1 - Operations Sort of Functioning"
+- "P2 - Operations Not Functioning"
+- "P3 - Operations Foobar (ciritical bug affecting master)"
```
Comment (whyrusleeping) [inline docs/github-issue-guide.md]: critical*

File: docs/github-issue-guide.md (hunk 2 above)
Comment (Kubuxu) [inline docs/github-issue-guide.md]: It is included in `ipfs version --all`, so is the platform.

Comment (whyrusleeping) [inline docs/github-issue-guide.md]: yeah, i think just including that output would be great

File: docs/github-issue-guide.md (hunk 3 above)
Comment (Kubuxu) [inline docs/github-issue-guide.md]: Heh, creative. `Foobar` rhymes with FUBAR.

**Comment by whyrusleeping** [comment]
alright, this roughly looks good to me.  Are each of these fields going to be tags? or just what were going to ask the user to fill in? If the latter, shouldnt we add this as an issue template?

cc @em-ly

**Comment by whyrusleeping** [comment]
Ah, missed the references from the issue template PR. LGTM :ship:

File: docs/github-issue-guide.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+- "tracking"
+- "unix vs dag"
+
+### Priority
+
+- "P0 - Operations Functioning"
+- "P1 - Operations Sort of Functioning"
+- "P2 - Operations Not Functioning"
+- "P3 - Operations Foobar (critical bug affecting master)"
+- "P4 - Operations on Fire (critical bug affecting release)"
```
Comment (jbenet) [inline docs/github-issue-guide.md]: Please reverse these

**Comment by jbenet** [comment]
happy this exists 👍

**Pull request merged**